packages/flutter_tools/lib/src/device.dart

```diff
@@ -423,6 +423,9 @@ abstract class Device {
   /// Whether the device supports the '--fast-start' development mode.
   bool get supportsFastStart => false;
 
+  /// Whether the device is a web-platform device.
+  bool get isWebDevice => false;
```

Comment:
Instead of adding isWebDevice you could check whether the target platform is web_javascript

Reply:
Fixed.